bin/build-package.js

```diff
@@ -0,0 +1,28 @@
+/** @format */
+
+// find all the packages
+const fs = require( 'fs' );
+const path = require( 'path' );
+const { execSync } = require( 'child_process' );
+
+const dir = process.cwd();
+const root = path.dirname( __dirname, '..' );
+
+const inputDir = path.join( dir, 'src' );
+const outputDirEsm = path.join( dir, 'dist', 'esm' );
+const outputDirCommon = path.join( dir, 'dist', 'cjs' );
+if ( fs.existsSync( inputDir ) ) {
+	console.log( 'Building %s', dir );
+	execSync( `node_modules/.bin/babel -d ${ outputDirEsm } ${ inputDir }`, {
+		env: Object.assign( {}, process.env, { CALYPSO_CLIENT: 'true' } ),
+		cwd: root,
+	} );
+	execSync( `node_modules/.bin/babel -d ${ outputDirCommon } ${ inputDir }`, {
+		env: Object.assign( {}, process.env, { CALYPSO_CLIENT: 'false' } ),
+		cwd: root,
+	} );
+} else {
+	console.log( 'Skipping %s (no src directory)', dir );
```

Comment:
can drop this branch

Reply:
43a0c76f3ee9f20d89958ecf1202c861514e889e